### PR TITLE
Clean up configuration options removing internal references

### PR DESCRIPTION
**Pull request by maddalab**
Summary:
Remove references to internal servers and update the readme
to make it clear, options must be updated for production use.

We should probably include a deployment README file

@defect @CaptPhunkosis @roymarantz @byxorna

### CODE REVIEW

File: testenv/README.md [hunk 1]
```diff
@@ -34,11 +34,20 @@ You can customize these in
...
 
-and modify settings as desired.
+and modify settings as desired. While the current values of the options
+allow you to proceed in the test environment without updates by executing
+```vagrant up```, you MUST provide production settings for
+
+* a DHCP server,
+* a TFTP server,
+* and a file server (serving static files over HTTP)
+* RPM repository that includes RPMs required by Genesis tasks
```
Comment (roymarantz) [inline testenv/README.md]: What is this about?  The testenv come self contained.  It provides all those services.  You need to provide them for your production environment which is already mentioned in INSTALL.md.  Of course that could be reworded if it isn't clear to people.

File: .gitignore [hunk 2]
```diff
@@ -11,3 +11,7 @@ Gemfile.lock
 # emacs
 *~
 \#*\#
+
+# built rpms
+bootcd/rpms/genesis_scripts-0.5-3.el6.noarch.rpm
+bootcd/rpms/genesis_scripts/genesis_scripts-0.5-3.el6.src.rpm
```
Comment (roymarantz) [inline .gitignore]: Why not ignore genesis_scripts-*.rpm instead of the specific version?

File: testenv/bootbox/web/config/config.yml [hunk 3]
```diff
@@ -5,24 +5,20 @@
         :kernel: /ipxe-images/genesis-vmlinuz
     :kernel_flags: rootflags=loop root=live:/genesis.iso rootfstype=auto ro liveimg vga=791 rd_NO_LUKS rd_NO_MD rd_NO_DM console=tty0 console=ttyS1,115200
     :menu_default: intake
-    :image_server: http://nexus.ewr01.tumblr.net:8888/genesis
+    :image_server: http://nexus.example.com:8888/genesis
```
Comment (roymarantz) [inline testenv/bootbox/web/config/config.yml]: IMO, this needs a # FIXME comment.

File: testenv/bootbox/web/config/config.yml [hunk 4]
```diff
@@ -5,24 +5,20 @@
...
-:gem:
-    :http_proxy: http://
-    :source: http://
+:tasks_url: http://192.168.33.10:8888/tasks
+:yum_repos:
+    :epel:
+        :label:   "[epel]"
+        :name:    "epel"
+        :descr:   "Epel"
+        :baseurl: "http://repo.example.com/mrepo/RPMS.epel"
```
Comment (roymarantz) [inline testenv/bootbox/web/config/config.yml]: # FIXME

File: testenv/README.md (hunk 1 above)
Comment (maddalab) [inline testenv/README.md]: I think this is harmless and probably a good idea to repeat. Leaving this in here for now

**Comment by maddalab** [comment]
@roymarantz  I created a new ticket for the handful of properties we spoke about that were legacy and no longer used

https://github.com/tumblr/genesis/issues/25

**Pull request merged**